geode-core/src/main/java/org/apache/geode/internal/statistics/StatMonitorHandler.java

```diff
@@ -110,7 +100,7 @@ public void sampled(long nanosTimeStamp, List<ResourceInstance> resourceInstance
   }
 
   private void monitor(final long sampleTimeMillis, final List<ResourceInstance> resourceInstance) {
-    List<StatisticsMonitor> currentMonitors = StatMonitorHandler.this.monitors;
+    ConcurrentHashSet<StatisticsMonitor> currentMonitors = StatMonitorHandler.this.monitors;
```

Comment:
I don't think this copy is needed since `monitors` is never updated after initialization.